test/nightly/getRotatingValue.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+let locationCount: number = getStartingIndex();
+const locations: string[] = ['Australia East', 'Australia Southeast', 'Brazil South', 'Canada Central', 'Central US', 'East Asia', 'East US', 'East US 2', 'France Central', 'Japan East', 'Japan West', 'Korea Central', 'Korea South', 'North Europe', 'South Central US', 'Southeast Asia', 'UK South', 'West Europe', 'West US', 'West US 2'];
```

Comment:
I would shorten this list a bit - we want a big enough list to know we work in multiple regions, but we don't need to test every region. Sometimes lesser-used regions might get a feature later than others or sometimes a region could just have an outage. The platform team owns that and it's on them to test every region, not us.

With a smaller list, it's easier for us to debug issues if there's a failure - aka it's easier for us to know if the issue is on our side or the platform team's side.

Reply:
@ejizba Do you have any suggestions for the locations we need to cover? Or we can have the same choice as https://github.com/microsoft/vscode-azurefunctions/blob/8525f5e03ac943a79f5b1918f33d9f0ef3aeb519/test/nightly/getRotatingValue.ts#L14??

Reply:
Yeah the same locations would be fine.